man/calculateROCMeasures.Rd

```diff
@@ -80,3 +80,5 @@ Other performance: \code{\link{ConfusionMatrix}},
   \code{\link{performance}}, \code{\link{setAggregation}},
   \code{\link{setMeasurePars}}
 }
+\concept{performance}
```

Comment:
For some reason these are being added? Is this because I am on the newest version of roxygen or is this an old thing?